src/components/Column/Column.tsx

```diff
@@ -100,7 +100,14 @@ export const Column = ({id, name, color, visible, index}: ColumnProps) => {
     columnNameMode === "VIEW" ? (
       <div className={classNames("column__header-text-wrapper", {"column__header-text-wrapper--hidden": !visible})}>
         {!visible && <HiddenIcon className="column__header-hidden-icon" title={t("Column.hiddenColumn")} onClick={toggleVisibilityHandler} />}
-        <h2 onDoubleClick={() => setColumnNameMode("EDIT")} className={classNames("column__header-text", {"column__header-text--hidden": !visible})}>
+        <h2
+          onDoubleClick={() => {
+            if (isModerator) {
+              setColumnNameMode("EDIT");
```

Comment:
Your PR fixes the use but I don't like the `setColumnNameMode` function. It can only be one of `VIEW` or `EDIT` and therefore of type `boolean`. :)
You can leave it there for now though.